org.electronjs.Electron2.BaseApp.yml

```diff
@@ -94,3 +94,46 @@ modules:
         x-checker-data:
           type: git
           tag-pattern: ^v([\d.]+)$
+
+  # Chromium dynamically loads the Speech Dispatcher library libspeechd.so.2.
+  # If present, and --enable-speech-dispatcher flag is passed, Web Speech API will be enabled.
+  # This section is borrowed and tweaked from:
+  # https://github.com/flathub/net.lutris.Lutris/blob/76e94a0b80ef3ebc1b9a6b61f47d736f5ddd772c/net.lutris.Lutris.yml#L495
+  # https://gitlab.archlinux.org/archlinux/packaging/packages/speech-dispatcher/-/blob/414baaf78b5fe416df88a89ac18d2dd579a0c653/PKGBUILD
+  - name: libspeechd
+    config-opts:
+      - --disable-static
+      - --with-ibmtts=no
+      - --with-kali=no
+      - --with-baratinoo=no
+      - --with-voxin=no
+      - --without-flite
+      - --disable-python
+      - --with-systemdsystemunitdir=/app/lib/systemd/system/speech-dispatcherd.service
+    no-make-install: true
+    post-install:
+      - cd ./src/api/c && make install
```

Comment:
This should not be needed.

Reply:
This builds specifically the C API, the main artifact of which is `libspeechd.so`. This is opposed to building the entire speech-dispatcher suite which includes the daemon we aren't using.

We're building a subset of speech-dispatcher here. Should we instead build everything, and just add more entries to the cleanup section?

Reply:
Hmm seems fine, I didn't see the directory.